internal/install/ux/prompt_ui_prompter.go

```diff
@@ -58,10 +59,12 @@ func validateYesNo(msg string) error {
 }
 
 func (p *PromptUIPrompter) MultiSelect(msg string, options []string) ([]string, error) {
+	defaults := utils.MakeRange(0, len(options)-1)
 	selected := []string{}
 	prompt := &survey.MultiSelect{
 		Message: msg,
 		Options: options,
+		Default: defaults,
```

Comment:
Oh good, I'm glad this was easy.

Reply:
Me too, I thought we were going to need a fork.

Reply:
This part looks good. The rest of the PR, I'm unclear as to why those changes are needed. Given the screenshots you provided it looks to behave correctly.